apps/frontend/src/libs/components/table/table.tsx

```diff
@@ -48,8 +55,19 @@ const Table = <T extends object>({
 						table.getRowModel().rows.map((row) => (
 							<tr className={styles["table-row"]} key={row.id}>
 								{row.getVisibleCells().map((cell) => (
-									<td className={styles["table-data"]} key={cell.id}>
-										{flexRender(cell.column.columnDef.cell, cell.getContext())}
+									<td
+										className={getValidClassNames(
+											styles["table-data"],
+											cell.column.id === "options" && styles["options-column"],
+										)}
+										key={cell.id}
+									>
+										{typeof cell.getValue() === "object"
+											? (cell.getValue() as React.ReactNode)
+											: flexRender(
+													cell.column.columnDef.cell,
+													cell.getContext(),
```

Comment:
What is this for?

Reply:
`flexRender` doesn't display react components, so we have to check what the content of cell